2024-08-21 13:12:46,150 INFO    StreamThr :68569 [internal.py:wandb_internal():85] W&B internal server running at pid: 68569, started at: 2024-08-21 13:12:46.149898
2024-08-21 13:12:46,152 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: status
2024-08-21 13:12:46,166 INFO    WriterThread:68569 [datastore.py:open_for_write():87] open: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/run-jnx0tyh2.wandb
2024-08-21 13:12:46,167 DEBUG   SenderThread:68569 [sender.py:send():391] send: header
2024-08-21 13:12:46,210 DEBUG   SenderThread:68569 [sender.py:send():391] send: run
2024-08-21 13:12:46,981 INFO    SenderThread:68569 [dir_watcher.py:__init__():211] watching files in: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files
2024-08-21 13:12:46,981 INFO    SenderThread:68569 [sender.py:_start_run_threads():1200] run started: jnx0tyh2 with start time 1724238766.151213
2024-08-21 13:12:46,988 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: check_version
2024-08-21 13:12:46,988 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: check_version
2024-08-21 13:12:47,064 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: run_start
2024-08-21 13:12:47,069 DEBUG   HandlerThread:68569 [system_info.py:__init__():26] System info init
2024-08-21 13:12:47,069 DEBUG   HandlerThread:68569 [system_info.py:__init__():41] System info init done
2024-08-21 13:12:47,069 INFO    HandlerThread:68569 [system_monitor.py:start():194] Starting system monitor
2024-08-21 13:12:47,069 INFO    SystemMonitor:68569 [system_monitor.py:_start():158] Starting system asset monitoring threads
2024-08-21 13:12:47,070 INFO    HandlerThread:68569 [system_monitor.py:probe():214] Collecting system info
2024-08-21 13:12:47,070 INFO    SystemMonitor:68569 [interfaces.py:start():188] Started cpu monitoring
2024-08-21 13:12:47,070 INFO    SystemMonitor:68569 [interfaces.py:start():188] Started disk monitoring
2024-08-21 13:12:47,073 INFO    SystemMonitor:68569 [interfaces.py:start():188] Started gpuapple monitoring
2024-08-21 13:12:47,075 INFO    SystemMonitor:68569 [interfaces.py:start():188] Started memory monitoring
2024-08-21 13:12:47,075 INFO    SystemMonitor:68569 [interfaces.py:start():188] Started network monitoring
2024-08-21 13:12:47,174 ERROR   HandlerThread:68569 [gpu_apple.py:probe():176] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 164, in probe
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:12:47,176 DEBUG   HandlerThread:68569 [system_info.py:probe():152] Probing system
2024-08-21 13:12:47,181 DEBUG   HandlerThread:68569 [system_info.py:_probe_git():137] Probing git
2024-08-21 13:12:47,208 DEBUG   HandlerThread:68569 [system_info.py:_probe_git():145] Probing git done
2024-08-21 13:12:47,208 DEBUG   HandlerThread:68569 [system_info.py:probe():200] Probing system done
2024-08-21 13:12:47,208 DEBUG   HandlerThread:68569 [system_monitor.py:probe():223] {'os': 'macOS-14.5-arm64-arm-64bit', 'python': '3.11.5', 'heartbeatAt': '2024-08-21T11:12:47.176860', 'startedAt': '2024-08-21T11:12:46.139548', 'docker': None, 'cuda': None, 'args': (), 'state': 'running', 'program': '<python with no main file>', 'codePathLocal': None, 'git': {'remote': 'https://github.com/jackbmontgomery/deep-predictive-active-inference.git', 'commit': '1691130b1ebb3fdf0ed5eb01f8f3d39f97f44a07'}, 'email': 'jackbmontgomery2001@gmail.com', 'root': '/Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference', 'host': 'Jacks-MacBook-Pro-2.local', 'username': 'jackmontgomery', 'executable': '/Users/jackmontgomery/anaconda3/bin/python', 'cpu_count': 8, 'cpu_count_logical': 8, 'disk': {'/': {'total': 228.27386474609375, 'used': 212.72147750854492}}, 'gpuapple': {'type': 'arm', 'vendor': 'Apple'}, 'memory': {'total': 16.0}}
2024-08-21 13:12:47,208 INFO    HandlerThread:68569 [system_monitor.py:probe():224] Finished collecting system info
2024-08-21 13:12:47,208 INFO    HandlerThread:68569 [system_monitor.py:probe():227] Publishing system info
2024-08-21 13:12:47,208 DEBUG   HandlerThread:68569 [system_info.py:_save_conda():209] Saving list of conda packages installed into the current environment
2024-08-21 13:12:47,250 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:12:47,982 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/conda-environment.yaml
2024-08-21 13:12:49,347 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:12:50,640 DEBUG   HandlerThread:68569 [system_info.py:_save_conda():224] Saving conda packages done
2024-08-21 13:12:50,641 INFO    HandlerThread:68569 [system_monitor.py:probe():229] Finished publishing system info
2024-08-21 13:12:50,643 DEBUG   SenderThread:68569 [sender.py:send():391] send: files
2024-08-21 13:12:50,643 INFO    SenderThread:68569 [sender.py:_save_file():1466] saving file wandb-metadata.json with policy now
2024-08-21 13:12:50,919 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: python_packages
2024-08-21 13:12:50,919 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: python_packages
2024-08-21 13:12:50,920 DEBUG   SenderThread:68569 [sender.py:send():391] send: telemetry
2024-08-21 13:12:50,920 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:12:50,920 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: stop_status
2024-08-21 13:12:50,920 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: stop_status
2024-08-21 13:12:50,989 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/conda-environment.yaml
2024-08-21 13:12:50,989 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/wandb-metadata.json
2024-08-21 13:12:50,989 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/requirements.txt
2024-08-21 13:12:51,389 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: status_report
2024-08-21 13:12:51,451 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:12:51,880 INFO    wandb-upload_0:68569 [upload_job.py:push():130] Uploaded file /var/folders/qc/h4k546bs6j9bl76bcjwyw5cw0000gn/T/tmpzkx6dolfwandb/334joa0s-wandb-metadata.json
2024-08-21 13:12:51,920 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:12:51,991 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:12:52,925 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:12:53,569 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:12:53,927 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:12:53,997 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:12:54,928 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:12:55,684 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:12:55,929 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:12:56,927 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: status_report
2024-08-21 13:12:56,931 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:12:57,791 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:12:57,934 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:12:58,014 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:12:58,936 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:12:59,896 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:12:59,938 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:00,019 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:13:00,943 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:01,946 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:02,111 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:13:02,734 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: status_report
2024-08-21 13:13:02,948 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:03,949 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:04,037 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:13:04,221 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:13:04,954 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:05,924 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: stop_status
2024-08-21 13:13:05,924 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: stop_status
2024-08-21 13:13:05,959 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:06,048 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:13:06,340 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:13:06,965 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:07,970 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:08,058 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:13:08,290 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: status_report
2024-08-21 13:13:08,447 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:13:08,972 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:09,975 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:10,557 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:13:10,976 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:11,981 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:12,071 INFO    Thread-12 :68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:13:12,670 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:13:12,982 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:13,985 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:14,276 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: status_report
2024-08-21 13:13:14,781 ERROR   gpuapple  :68569 [gpu_apple.py:sample():110] GPU stats error: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
Traceback (most recent call last):
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/sdk/internal/system/assets/gpu_apple.py", line 64, in sample
    subprocess.check_output(command, universal_newlines=True)
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 466, in check_output
    return run(*popenargs, stdout=PIPE, timeout=timeout, check=True,
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/Users/jackmontgomery/anaconda3/lib/python3.11/subprocess.py", line 571, in run
    raise CalledProcessError(retcode, process.args,
subprocess.CalledProcessError: Command '['/Users/jackmontgomery/anaconda3/lib/python3.11/site-packages/wandb/bin/apple_gpu_stats', '--json']' died with <Signals.SIGTRAP: 5>.
2024-08-21 13:13:14,816 DEBUG   SenderThread:68569 [sender.py:send():391] send: telemetry
2024-08-21 13:13:14,816 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: server_info
2024-08-21 13:13:14,816 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: server_info
2024-08-21 13:13:15,108 DEBUG   SenderThread:68569 [sender.py:send():391] send: exit
2024-08-21 13:13:15,108 INFO    SenderThread:68569 [sender.py:send_exit():598] handling exit code: 0
2024-08-21 13:13:15,108 INFO    SenderThread:68569 [sender.py:send_exit():600] handling runtime: 28
2024-08-21 13:13:15,109 INFO    SenderThread:68569 [sender.py:_save_file():1466] saving file wandb-summary.json with policy end
2024-08-21 13:13:15,109 INFO    SenderThread:68569 [sender.py:send_exit():606] send defer
2024-08-21 13:13:15,109 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,109 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 0
2024-08-21 13:13:15,109 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,109 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 0
2024-08-21 13:13:15,109 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 1
2024-08-21 13:13:15,109 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,109 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 1
2024-08-21 13:13:15,110 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,110 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 1
2024-08-21 13:13:15,110 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 2
2024-08-21 13:13:15,110 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,110 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 2
2024-08-21 13:13:15,110 INFO    HandlerThread:68569 [system_monitor.py:finish():203] Stopping system monitor
2024-08-21 13:13:15,110 DEBUG   SystemMonitor:68569 [system_monitor.py:_start():172] Starting system metrics aggregation loop
2024-08-21 13:13:15,110 DEBUG   SystemMonitor:68569 [system_monitor.py:_start():179] Finished system metrics aggregation loop
2024-08-21 13:13:15,110 DEBUG   SystemMonitor:68569 [system_monitor.py:_start():183] Publishing last batch of metrics
2024-08-21 13:13:15,110 INFO    HandlerThread:68569 [interfaces.py:finish():200] Joined cpu monitor
2024-08-21 13:13:15,111 INFO    HandlerThread:68569 [interfaces.py:finish():200] Joined disk monitor
2024-08-21 13:13:15,111 INFO    HandlerThread:68569 [interfaces.py:finish():200] Joined gpuapple monitor
2024-08-21 13:13:15,111 INFO    HandlerThread:68569 [interfaces.py:finish():200] Joined memory monitor
2024-08-21 13:13:15,111 INFO    HandlerThread:68569 [interfaces.py:finish():200] Joined network monitor
2024-08-21 13:13:15,112 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,112 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 2
2024-08-21 13:13:15,112 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 3
2024-08-21 13:13:15,112 DEBUG   SenderThread:68569 [sender.py:send():391] send: telemetry
2024-08-21 13:13:15,112 DEBUG   SenderThread:68569 [sender.py:send():391] send: stats
2024-08-21 13:13:15,112 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,112 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 3
2024-08-21 13:13:15,112 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,112 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 3
2024-08-21 13:13:15,112 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 4
2024-08-21 13:13:15,112 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,112 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 4
2024-08-21 13:13:15,112 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,112 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 4
2024-08-21 13:13:15,112 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 5
2024-08-21 13:13:15,112 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,112 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 5
2024-08-21 13:13:15,112 DEBUG   SenderThread:68569 [sender.py:send():391] send: summary
2024-08-21 13:13:15,113 INFO    SenderThread:68569 [sender.py:_save_file():1466] saving file wandb-summary.json with policy end
2024-08-21 13:13:15,113 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,113 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 5
2024-08-21 13:13:15,113 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 6
2024-08-21 13:13:15,113 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,113 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 6
2024-08-21 13:13:15,113 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,113 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 6
2024-08-21 13:13:15,115 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: status_report
2024-08-21 13:13:15,434 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 7
2024-08-21 13:13:15,435 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,435 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 7
2024-08-21 13:13:15,435 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,435 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 7
2024-08-21 13:13:15,571 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 8
2024-08-21 13:13:15,571 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,571 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 8
2024-08-21 13:13:15,572 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,572 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 8
2024-08-21 13:13:15,573 INFO    SenderThread:68569 [job_builder.py:build():440] Attempting to build job artifact
2024-08-21 13:13:15,573 INFO    SenderThread:68569 [job_builder.py:_get_source_type():569] is repo sourced job
2024-08-21 13:13:15,573 INFO    SenderThread:68569 [job_builder.py:_get_program_relpath():587] run is notebook based run
2024-08-21 13:13:15,574 WARNING SenderThread:68569 [job_builder.py:_log_if_verbose():274] Source type is set to 'repo' but some required information is missing from the environment. A job will not be created from this run. See https://docs.wandb.ai/guides/launch/create-job
2024-08-21 13:13:15,574 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 9
2024-08-21 13:13:15,574 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:15,574 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 9
2024-08-21 13:13:15,574 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:15,574 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 9
2024-08-21 13:13:15,574 INFO    SenderThread:68569 [dir_watcher.py:finish():358] shutting down directory watcher
2024-08-21 13:13:16,085 INFO    SenderThread:68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:13:16,085 INFO    SenderThread:68569 [dir_watcher.py:_on_file_modified():288] file/dir modified: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/config.yaml
2024-08-21 13:13:16,085 INFO    SenderThread:68569 [dir_watcher.py:_on_file_created():271] file/dir created: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/wandb-summary.json
2024-08-21 13:13:16,085 INFO    SenderThread:68569 [dir_watcher.py:finish():388] scan: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files
2024-08-21 13:13:16,085 INFO    SenderThread:68569 [dir_watcher.py:finish():402] scan save: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/requirements.txt requirements.txt
2024-08-21 13:13:16,086 INFO    SenderThread:68569 [dir_watcher.py:finish():402] scan save: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log output.log
2024-08-21 13:13:16,087 INFO    SenderThread:68569 [dir_watcher.py:finish():402] scan save: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/config.yaml config.yaml
2024-08-21 13:13:16,089 INFO    SenderThread:68569 [dir_watcher.py:finish():402] scan save: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/wandb-summary.json wandb-summary.json
2024-08-21 13:13:16,090 INFO    SenderThread:68569 [dir_watcher.py:finish():402] scan save: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/conda-environment.yaml conda-environment.yaml
2024-08-21 13:13:16,092 INFO    SenderThread:68569 [dir_watcher.py:finish():402] scan save: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/wandb-metadata.json wandb-metadata.json
2024-08-21 13:13:16,092 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 10
2024-08-21 13:13:16,093 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:16,094 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 10
2024-08-21 13:13:16,095 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:16,095 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 10
2024-08-21 13:13:16,095 INFO    SenderThread:68569 [file_pusher.py:finish():169] shutting down file pusher
2024-08-21 13:13:16,110 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: poll_exit
2024-08-21 13:13:16,110 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: poll_exit
2024-08-21 13:13:16,797 INFO    wandb-upload_0:68569 [upload_job.py:push():130] Uploaded file /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/requirements.txt
2024-08-21 13:13:17,004 INFO    wandb-upload_2:68569 [upload_job.py:push():130] Uploaded file /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/config.yaml
2024-08-21 13:13:17,118 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: poll_exit
2024-08-21 13:13:17,119 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: poll_exit
2024-08-21 13:13:17,154 INFO    wandb-upload_1:68569 [upload_job.py:push():130] Uploaded file /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/output.log
2024-08-21 13:13:17,154 INFO    wandb-upload_3:68569 [upload_job.py:push():130] Uploaded file /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/wandb-summary.json
2024-08-21 13:13:17,253 INFO    wandb-upload_4:68569 [upload_job.py:push():130] Uploaded file /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/files/conda-environment.yaml
2024-08-21 13:13:17,458 INFO    Thread-11 (_thread_body):68569 [sender.py:transition_state():626] send defer: 11
2024-08-21 13:13:17,458 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:17,458 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 11
2024-08-21 13:13:17,458 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:17,458 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 11
2024-08-21 13:13:17,459 INFO    SenderThread:68569 [file_pusher.py:join():175] waiting for file pusher
2024-08-21 13:13:17,459 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 12
2024-08-21 13:13:17,459 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:17,460 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 12
2024-08-21 13:13:17,460 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:17,460 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 12
2024-08-21 13:13:17,460 INFO    SenderThread:68569 [file_stream.py:finish():601] file stream finish called
2024-08-21 13:13:18,099 INFO    SenderThread:68569 [file_stream.py:finish():605] file stream finish is done
2024-08-21 13:13:18,099 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 13
2024-08-21 13:13:18,099 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:18,099 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 13
2024-08-21 13:13:18,099 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:18,099 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 13
2024-08-21 13:13:18,099 INFO    SenderThread:68569 [sender.py:transition_state():626] send defer: 14
2024-08-21 13:13:18,099 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: defer
2024-08-21 13:13:18,099 INFO    HandlerThread:68569 [handler.py:handle_request_defer():184] handle defer: 14
2024-08-21 13:13:18,099 DEBUG   SenderThread:68569 [sender.py:send():391] send: final
2024-08-21 13:13:18,099 DEBUG   SenderThread:68569 [sender.py:send():391] send: footer
2024-08-21 13:13:18,099 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: defer
2024-08-21 13:13:18,099 INFO    SenderThread:68569 [sender.py:send_request_defer():622] handle sender defer: 14
2024-08-21 13:13:18,100 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: poll_exit
2024-08-21 13:13:18,100 DEBUG   SenderThread:68569 [sender.py:send_request():418] send_request: poll_exit
2024-08-21 13:13:18,100 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: internal_messages
2024-08-21 13:13:18,101 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: get_summary
2024-08-21 13:13:18,102 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: sampled_history
2024-08-21 13:13:18,104 DEBUG   HandlerThread:68569 [handler.py:handle_request():158] handle_request: shutdown
2024-08-21 13:13:18,104 INFO    HandlerThread:68569 [handler.py:finish():882] shutting down handler
2024-08-21 13:13:19,101 INFO    SenderThread:68569 [sender.py:finish():1631] shutting down sender
2024-08-21 13:13:19,101 INFO    SenderThread:68569 [file_pusher.py:finish():169] shutting down file pusher
2024-08-21 13:13:19,101 INFO    SenderThread:68569 [file_pusher.py:join():175] waiting for file pusher
2024-08-21 13:13:19,101 INFO    WriterThread:68569 [datastore.py:close():296] close: /Users/jackmontgomery/Desktop/UCT/Research_Project/deep-predictive-active-inference/wandb/run-20240821_131246-jnx0tyh2/run-jnx0tyh2.wandb
